acvm-repo/blackbox_solver/src/bigint.rs

```diff
@@ -97,3 +97,51 @@ impl BigIntSolver {
         Ok(())
     }
 }
+
+/// Wrapper over the generic bigint solver to automatically assign bigint IDs.
+#[derive(Default, Debug, Clone, PartialEq, Eq)]
+pub struct BigintSolverWithId {
```

Comment:
Just for consistency with the camel case can we rename this symbol to `BigIntSolverWithId`? This looks like it was moved from `acvm-repo/brillig_vm/src/black_box.rs` which also had `Bigint`. Let's just rename it here.

Reply:
Renamed both.